README.md

```diff
@@ -12,29 +12,41 @@ applications for [Tendermint](https://github.com/tendermint/tendermint/).
 
 ## Supported Version
 
-- Tendermint 0.32.7
-- ABCI 0.16.0
+- Tendermint 0.33.0
+- ABCI 0.16.1
 
-## Installation
+## Usage
 
-### Dependencies
+Add `abci` in your `Cargo.toml`'s `dependencies` section:
 
-Make sure that you have Rust and Cargo installed. The easiest way is to follow the instructions on [rustup](https://rustup.rs/).
+```toml
+[dependencies]
+abci = "0.7"
+```
```

Comment:
I guess one also needs to add:
```
async-trait = "0.1"
```
?

Reply:
This is not needed because we re-export `async_trait` macro.